.github/workflows/build-latest.yaml

```diff
@@ -19,5 +19,5 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm ci
+      - run: npm install
```

Comment:
I really don't see the point of this. Dropping the package-lock.json and the usage om npm ci means that the "build" will no longer be reproducible